app/apps.json

```diff
@@ -325,5 +325,15 @@
       "type": "addon",
       "url": "https://github.com/justindarc/fxos-animated-clock",
       "revision": "955cf91115a1a7dc4baf1ae6d6f9166bf3184322"
+    },
+    "https://justindarc.github.io/fxos-animated-clock/manifest.webapp": {
+      "manifestURL": "https://raw.githubusercontent.com/novia713/monchi-dificil/master/manifest.webapp",
```

Comment:
Better is to host this on gh-pages. See here for information on how to do that:

https://help.github.com/articles/creating-project-pages-manually/